crypto/asn1/a_time.c

```diff
@@ -489,7 +489,7 @@ int ASN1_TIME_print(BIO *bp, const ASN1_TIME *tm)
         if (tm->length > 15 && v[14] == '.') {
             f = &v[14];
             f_len = 1;
-            while (14 + f_len < l && isdigit(f[f_len]))
+            while (14 + f_len < l && ossl_isdigit(f[f_len]))
```

Comment:
I believe that all these should be always be ASCII comparisons given that this is an ASN1 string... but I noticed the comment on dropping the EBCDIC tables, so I guess I'd like to see a proper statement of work in the description (or at least in a comment here). Basically, how is EBCDIC handled?